src/components/ErrorShow/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './ErrorShow';
```

Comment:
It's recommended to explicitly name the exported module for better error stack trace. You can update the export statement like this: `export { ErrorShow } from './ErrorShow';`